src/main/java/io/vlingo/xoom/actors/Actor.java

```diff
@@ -194,7 +194,9 @@ protected Actor() {
    */
   protected <R> Completes<R> answerFrom(final Completes<R> eventualOutcome) {
     final CompletesEventually completes = completesEventually();
-    eventualOutcome.andFinallyConsume((R value) -> completes.with(value));
+    eventualOutcome
+            .otherwiseConsume((R value) -> completes.with(value))
```

Comment:
I also tried `otherwise((R value) -> value)` here and the test passed. However, only `otherwiseConsume()` actually worked in schemata.